ci/Jenkinsfile_utils.groovy

```diff
@@ -144,9 +157,78 @@ def docker_run(platform, function_name, use_nvidia, shared_mem = '500m') {
   sh command
 }
 
+// Allow publishing to GitHub with a custom context (the status shown under a PR)
+// Credit to https://plugins.jenkins.io/github
+def get_repo_url() {
+  checkout scm
+  sh "git config --get remote.origin.url > .git/remote-url"
```

Comment:
Why not .execute()  writing a file doesn't seem the best way.

Reply:
There is no way to directly access shell output in Jenkins https://stackoverflow.com/questions/36507410/is-it-possible-to-capture-the-stdout-from-the-sh-dsl-command-in-the-pipeline

I could've used other methods, but that would then have triggered the security mechanisms of Jenkins due to the sandboxing mechanism. Execute (although I don't know exactly which function you're referring to) sounds like it would probably trigger that restriction.